bitnami/vault/templates/injector/service.yaml

```diff
@@ -57,7 +49,8 @@ spec:
     {{- if .Values.injector.service.extraPorts }}
     {{- include "common.tplvalues.render" (dict "value" .Values.injector.service.extraPorts "context" $) | nindent 4 }}
     {{- end }}
-  selector: {{- include "common.labels.matchLabels" . | nindent 4 }}
+  {{- $podLabels := merge .Values.injector.podLabels .Values.commonLabels }}
```

Comment:
Performed a manual check and confirmed that the `selector` field in a service is mutable, so there should be no issues during upgrades.